Core/OrderHelper.php

```diff
@@ -10,36 +10,39 @@
 namespace Wirecard\Oxid\Core;
 
 use Exception;
-use OxidEsales\Eshop\Core\Field;
-use OxidEsales\Eshop\Core\Registry;
 use OxidEsales\Eshop\Application\Model\Basket;
+use OxidEsales\Eshop\Application\Model\User;
 use OxidEsales\Eshop\Core\Exception\ArticleInputException;
 use OxidEsales\Eshop\Core\Exception\NoArticleException;
 use OxidEsales\Eshop\Core\Exception\OutOfStockException;
-use OxidEsales\Eshop\Application\Model\User;
-
+use OxidEsales\Eshop\Core\Field;
```

Comment:
As you always slit up so nicely the uses please also group and separate them with a new line as in other files.